docs/guide/testing.md

```diff
@@ -7,30 +7,43 @@ This guide explains how to test components, which use Font Awesome icons.
 Testing components using explicit reference does not require any special setup. You only need to add `FontAwesomeModule` to the `imports` of the testing module to be able to use `angular-fontawesome` components in the tests.
 
 ```typescript
-import { Component } from '@angular/core';
-import { faUser } from '@fortawesome/free-solid-svg-icons';
+import { NgModule } from '@angular/core';
 
-@Component({
-  selector: 'app-explicit-reference',
-  template: '<fa-icon [icon]="faUser"></fa-icon>',
+import { FontAwesomeModule, FaIconLibrary, FaConfig } from '@fortawesome/angular-fontawesome';
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+import { faFacebook } from '@fortawesome/free-brands-svg-icons';
+
+@NgModule({
+  declarations: [],
+  providers: [FaIconLibrary, FaConfig],
+  imports: [FontAwesomeModule],
+  exports: [FontAwesomeModule]
 })
-export class ExplicitReferenceComponent {
-  faUser = faUser;
+export class FontAwesomeIconsModule {
+  constructor(library: FaIconLibrary) {
+    library.addIcons(
+      faStar,
+      faFacebook,
+    );
+  }
 }
 ```
 
 ```typescript
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
-import { ExplicitReferenceComponent } from './explicit-reference.component';
+import { FontAwesomeIconsModule } from './font-awesome-icons.module';
 
 describe('ExplicitReferenceComponent', () => {
   let component: ExplicitReferenceComponent;
   let fixture: ComponentFixture<ExplicitReferenceComponent>;
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [FontAwesomeModule], // <--
+      imports: [
+          FontAwesomeModule       // <--
```

Comment:
I don't think `FontAwesomeModule` needs to be imported as it is added to `exports` of the `FontAwesomeIconsModule`.